cmd/eupf/main.go

```diff
@@ -1,6 +1,8 @@
 package main
 
 import (
+	"github.com/edgecomllc/eupf/cmd/eupf/core"
```

Comment:
`eupf` apears twice in path `github.com/edgecomllc/eupf/cmd/eupf/core`. Looks redundant

Reply:
That how project was laid out from the start, than we need to move everything from cmd/eupf folder to cmd.

Reply:
Why not to move right now?